circle.yml

```diff
@@ -0,0 +1,18 @@
+machine:
+  timezone: America/Los_Angeles
+  php:
+    version: 5.3.3
+  environment:
+    COMPOSER_FLAGS: "--prefer-stable --prefer-lowest"
+
+dependencies:
+  override:
+    - mkdir -p build/logs
+    - composer update ${COMPOSER_FLAGS} --no-interaction
+    - sed -i 's/^;//' ~/.phpenv/versions/$(phpenv global)/etc/conf.d/xdebug.ini
+
+test:
+  override:
+    - phpunit -c travis.phpunit.xml
```

Comment:
It's going to be renamed to ci.phpunit.xml in another PR, but I'm OK with deleting it too.